README.md

```diff
@@ -1,5 +1,15 @@
 # Friction with Probot
 
+> A place to talk about any friction experienced using Probot.
+
 Where have you experienced frustration, confusion, annoyance, or any other unpleasant feels while using Probot? This repository is a place to document and discuss any friction with using Probot and building GitHub Apps.
 
 Please [open an issue](https://github.com/probot/friction/issues/new) and help us make Probot better.
+
+## Contribute
+
+This is a discussion repository concerning all of Probot; all of the activity here happens in [the issues](https://github.com/probot/friction/issues). That having been said, if you have a suggestion for how the flow or documentation in this repo could be improved, feel free to suggest something to us. Not sure how to write an issue, or have other questions? Check out our [Contributing](CONTRIBUTING.md) doc.
+
+Want to chat with Probot users and contributors before posting here? [Join us in Slack](https://probot-slackin.herokuapp.com/)!
+
+Like everywhere else in the Probot ecosystem, please conform to the [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
What would you think about linking to https://github.com/probot/probot/blob/master/CODE_OF_CONDUCT.md instead of duplicating this file across all repos?

Reply:
I think that works fine! We hadn't been doing that yet, but it's the smart thing to do. But I think it should actually point to a welcome repo, and I'm not sure that probot/probot actually is the best welcome repo. It might be better to rename probot/probot as probot/probotjs, and then institute a probot/welcome. @gr2m mention this here, too: https://github.com/probot/friction/issues/21

Reply:
Done. Thanks, @hiimbex.